src/webgpu/api/operation/device/lost.spec.ts

```diff
@@ -13,7 +14,14 @@ class DeviceLostTests extends Fixture {
   readonly kDeviceLostTimeoutMS = 2000;
 
   getDeviceLostWithTimeout(lost: Promise<GPUDeviceLostInfo>): Promise<GPUDeviceLostInfo> {
-    return raceWithRejectOnTimeout(lost, this.kDeviceLostTimeoutMS, 'device was not lost');
+    // Set `noRaceWithRejectOnTimeout` to false, and then restore it after calling
+    // `raceWithRejectOnTimeout`. Here, it is used to ensure a Promise is not resolved, not
+    // to ensure a Promise must settle by some deadline.
```

Comment:
I'm pretty sure only one of the tests (`not_lost_on_gc`) uses it in this way. The others do expect it to be resolved. Would it work to just update that one test? Here's a PR to do that: #1844